src/activities/ActivityUsageEntity.js

```diff
@@ -55,6 +55,23 @@ export class ActivityUsageEntity extends Entity {
 		return this._entity.getLinkByRel(Rels.Activities.activityCollection).href;
 	}
 
+	/**
+	 * @returns {bool} Whether the release conditions dialog opener sub entity is present.
+	 */
+	canEditReleaseConditions() {
+
+		return this._entity.hasSubEntityByRel(Rels.Activities.releaseConditionsDialogOpener);
+	}
+
+	/**
+	 * @returns {string} Url of the MVC release conditions dialog.
+	 */
+	editReleaseConditionsUrl() {
+
+		const entity = this._entity.getSubEntityByRel(Rels.Activities.releaseConditionsDialogOpener);
+		return entity ? entity.properties.url : undefined;
```

Comment:
You can do `entity && entity.properties && entity.properties.url` which will either return `undefined` or `entity.properties.url`